.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 8.0.x
+          global-json-file: global.json
```

Comment:
This may be work by default without explicitly specifying it.

Reply:
The docs don't indicate this but it is worth trying